ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx

```diff
@@ -264,6 +267,29 @@ const TopologyComponent = ({ detailType, detailId }: TopologyComponentProps) =>
             }
             sideBarOpen={!!selectedEntity}
             sideBarResizable
+            controlBar={
+                <TopologyControlBar
+                    controlButtons={createTopologyControlButtons({
+                        ...defaultControlButtonsOptions,
+                        zoomInCallback: () => {
+                            controller.getGraph().scaleBy(4 / 3);
+                        },
+                        zoomOutCallback: () => {
+                            controller.getGraph().scaleBy(0.75);
+                        },
```

Comment:
Damn irrational numbers!

Reply:
yeah i just copied from the demo... we can fudge around with this during polish, but i feel like it works fine for now